Api/V8/Dictionary/Response.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Api\V8\Dictionary;
+
+class Response
```

Comment:
Why do we need this class ?

Reply:
If someone wants to translate the errors, exceptions, etc in the future, this dir is for it.
Also, we can customize whatever we want, like error data.